docker/driver-loader-buster/Dockerfile

```diff
@@ -123,7 +123,12 @@ RUN curl -L -o binutils_2.30-22_${TARGETARCH}.deb https://download.falco.org/dep
 	&& curl -L -o binutils-common_2.30-22_${TARGETARCH}.deb https://download.falco.org/dependencies/binutils-common_2.30-22_${TARGETARCH}.deb \
 	&& dpkg -i *binutils*.deb \
 	&& rm -f *binutils*.deb
+	
+# Install latest falcoctl
+RUN curl -L -o falcoct.tar.gz $(curl -s "https://api.github.com/repos/falcosecurity/falcoctl/releases/latest" | jq -r '.assets[] | select(.name|test(".linux_$TARGETARCH.tar.gz")) | .browser_download_url') && tar -xvf falcoctl.tar.gz && mv falcoctl /usr/bin
```

Comment:
```suggestion
```

Since we installed Falco using .deb (which already includes `falcoctl`), why should we replace it with the latest version?

In the end, falcoctl and Falco versions provided by the deb package are guaranteed to work together, whereas the latest falcoctl version might not.

Unless I'm missing something, I'd say not to overwrite falcoctl.

Reply:
Makes sense!